DOCS.md

```diff
@@ -0,0 +1,58 @@
+<p align="center">
+  <a href="https://www.contentful.com/developers/docs/references/content-delivery-api/">
+    <img alt="Contentful Logo" title="Contentful" src="media://contentful-icon.png" width="150">
```

Comment:
For me the logo is not loading here: https://github.com/contentful/contentful.js/blob/chore/upgrade-to-v10/DOCS.md is that expected?

Reply:
it's for the generated typedoc, it's not meant to be rendered on github